cypress/e2e/support/utils/generateProducts.ts

```diff
@@ -7,7 +7,7 @@ export const generateProducts = (count: number, options: Partial<ProductCatalogI
     productsList.push({
       index: 'product-catalog',
       source: {
-        id: 'a5afb115-025e-4da1-a013-bf05b326e0a5' + i,
+        id: 'a5afb115-025e-4da1-a013-bf05b326e0a' + i,
```

Comment:
This will only generate a valid uuid for integers < 6, should we just use a uuid generator function here instead?

Reply:
Agreed with mike, probably makes more sense.